samples/arch/mpu/mpu_test/README.rst

```diff
@@ -1,11 +1,11 @@
 .. _mpu_test:
 
-Memory Protection Unit (MPU) TEST
+Memory Protection Unit (MPU)
 #################################
 
 Overview
 ********
-This test provides a set options to check the correct MPU configuration
+This project provides a set options to check the correct MPU configuration
```

Comment:
I think we usually use `application` or simply `app`.

Reply:
Changed it to application.